src/sentry/static/sentry/app/components/avatar/index.tsx

```diff
@@ -6,14 +6,15 @@ import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
 import ProjectAvatar from 'app/components/avatar/projectAvatar';
 import TeamAvatar from 'app/components/avatar/teamAvatar';
 import UserAvatar from 'app/components/avatar/userAvatar';
-import {Team, OrganizationSummary, AvatarProject} from 'app/types';
+import {Team, OrganizationSummary, AvatarProject, Member} from 'app/types';
 
 const BasicModelShape = PropTypes.shape({slug: PropTypes.string});
 
 type Props = {
   team?: Team;
   organization?: OrganizationSummary;
   project?: AvatarProject;
+  member?: Member;
 } & UserAvatar['props'];
 
 class Avatar extends React.Component<Props> {
```

Comment:
Could you remove the `static propTypes` on Line 21 since you're touching this file? 

I think we didn't delete that in the early days of TS conversion but it should be okay to delete now.